### PR TITLE
Dark theme on first load

### PR DESCRIPTION
**Pull request by kakashy**
This would change the `$colors` variables with a dark theme on the first page load only. It does not affect changes made later.

### CODE REVIEW

File: src/routes/index.svelte [hunk 1]
```diff
@@ -168,6 +169,18 @@
 	};
 
 	onMount(() => {
+		const darkMode = () => {
+			const dark: boolean = window.matchMedia('(prefers-color-scheme: dark').matches;
```
Comment (pilcrowOnPaper) [inline src/routes/index.svelte:173]: typo: should be window.matchMedia('(prefers-color-scheme: dark)').matches

Comment (pilcrowonpaper) [inline src/routes/index.svelte]: typo: should be `.matchMedia('(prefers-color-scheme: dark)')`

Comment (pilcrowonpaper) [inline src/routes/index.svelte]: Also, should we check if `window.matchMedia` exists?

**Review by pilcrowonpaper — CHANGES_REQUESTED** [review]
Looks like it's not working in preview (https://niagara-853gypyjh-pilcrowonpaper.vercel.app)

I think it's due to a typo. Except that, looks good

File: src/routes/index.svelte (hunk 1 above)
Comment (kakashy) [reply src/routes/index.svelte]: Cat might have been on top of my keyboard. Sorry, fixing.

Comment (kakashy) [reply src/routes/index.svelte]: Yeah, you're right. Fixing.

**Comment by pilcrowonpaper** [comment]
Thanks for contributing!

**Comment by kakashy** [comment]
> Thanks for contributing!

Aye! Thanks for this lite timer. It's really convenient.

**Pull request merged**